src/backend/base/langflow/graph/vertex/base.py

```diff
@@ -495,7 +495,8 @@ async def _build(
             )
         else:
             custom_component = self._custom_component
-            self._custom_component.set_event_manager(event_manager)
+            if hasattr(custom_component, "set_event_manager"):
```

Comment:
Or could check that `self._custom_component` is instance of `Component` ?

Reply:
@cbornet worked as well :)